src/BasisTheory.net/Sessions/Requests/AuthorizeSessionRequest.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using BasisTheory.net.Applications.Entities;
+using Newtonsoft.Json;
+
+public class AuthorizeSessionRequest
+{
+    [JsonProperty("nonce")]
+    [JsonPropertyName("nonce")]
+    public string Nonce { get; set; }
+
+    [JsonProperty("expires_at")]
+    [JsonPropertyName("expires_at")]
+    public DateTimeOffset? ExpiresAt { get; set; }
+        
+    [JsonProperty("permissions")]
+    [JsonPropertyName("permissions")]
+    public List<string>? Permissions { get; set; }
```

Comment:
This nullable `?` declaration on a reference type generates a few warnings since we don't seem to have nullable reference types enabled either in this project or on this class.

> The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.

We should either remove the nullable declaration or follow these instructions to satisfy the warning.

https://stackoverflow.com/questions/55492214/the-annotation-for-nullable-reference-types-should-only-be-used-in-code-within-a
https://learn.microsoft.com/en-us/dotnet/csharp/nullable-references#nullable-contexts

Reply:
To keep things consistent between this sdk and the[ js-sdk](https://github.com/Basis-Theory/basis-theory-js/pull/264/files#diff-7e5ccedd67ac05b19171e919cf28b8f014af16d52bbee59b5383e2449ab3f0c0R4), I chose to add `#nullable enable` to the AuthorizeSessionRequest.cs since permissions and rules are nullable.